.github/workflows/codestyle.yml

```diff
@@ -0,0 +1,26 @@
+name: Codestyle
```

Comment:
let's rename to Static Code Analysis (or sca.yml)
we do more than style in it

Reply:
Done